### PR TITLE
crowdin 2.0.11 (new formula)

### PR DESCRIPTION
**Pull request by ihor-popyk**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: FORMULA/crowdin.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Crowdin < Formula
+  desc "Crowdin-CLI is a command-line tool that allows to manage your resources"
+  homepage "https://support.crowdin.com/cli-tool/"
+  url "https://crowdin.com/downloads/crowdin-cli-2.0.11.zip"
+  sha256 "027925176948b8973ba1a9fab4f0c097bf49f2df0c23be20d79049bd7af4fbc3"
+
```
Comment (fxcoudert) [inline FORMULA/crowdin.rb]: This needs a `depends_on :java` (or `depends_on :java => "1.8+"` if there is a minimum version requirement).

Comment (fxcoudert) [inline FORMULA/crowdin.rb]: Add a `bottle :unneeded` line

File: FORMULA/crowdin.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  def install
+    bin.install "crowdin-cli.jar"
+    system "alias", "crowdin=\"'java -jar #{bin}/crowdin-cli.jar'\" >> ~/.bashrc"
+    system "alias", "crowdin=\"'java -jar #{bin}/crowdin-cli.jar'\" >> ~/.bash_profile"
```
Comment (fxcoudert) [inline FORMULA/crowdin.rb]: Do not modify the user's config files (`.bashrc` and `.bash_profile`). You can write a wrapper script, like the `scm-manager.rb` does for `scm-cli-client`.

File: FORMULA/crowdin.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  test do
+    system "java", "-jar", "#{bin}/crowdin-cli.jar"
```
Comment (fxcoudert) [inline FORMULA/crowdin.rb]: What is that testing? What is the expected output and return code?

File: FORMULA/crowdin.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+class Crowdin < Formula
+  desc "Crowdin-CLI is a command-line tool that allows to manage your resources"
```
Comment (fxcoudert) [inline FORMULA/crowdin.rb]: Reword the description to be `Command-line tool that allows to manage your resources`

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: FORMULA/crowdin.rb (hunk 3 above)
Comment (ihor-popyk) [inline FORMULA/crowdin.rb]: It just runs help command and expects general help information

Comment (fxcoudert) [inline FORMULA/crowdin.rb]: Can we do a test that does something “useful”/representative of real use?

File: FORMULA/crowdin.rb (hunk 4 above)
Comment (ihor-popyk) [inline FORMULA/crowdin.rb]: done

File: FORMULA/crowdin.rb (hunk 1 above)
Comment (ihor-popyk) [inline FORMULA/crowdin.rb]: done

Comment (ihor-popyk) [inline FORMULA/crowdin.rb]: done

File: FORMULA/crowdin.rb (hunk 2 above)
Comment (ihor-popyk) [inline FORMULA/crowdin.rb]: done

File: FORMULA/crowdin.rb (hunk 3 above)
Comment (ihor-popyk) [inline FORMULA/crowdin.rb]: This is a simple test to ensure that the program is installed and working correctly. The next version will add more tests.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @ihor-popyk for your contribution to Homebrew!

**Pull request merged**